src/components/header/styles.css

```diff
@@ -0,0 +1,19 @@
+.header-container {
+  padding: 2.3rem 12rem;
+  background: var(--kelley-green);
+  width: 100%;
+  display: grid;
+  grid-template-columns: 16.5rem 1fr 9rem;
+  grid-template-areas: 'logo . logout';
```

Comment:
there's no need to have an empty middle column. You could have just added 2 columns and positioned the elements within them